docs/toolkit/ash-cli/tutorials/local-subnet.md

```diff
@@ -0,0 +1,256 @@
+```
+
+  </TabItem>
+  <TabItem value="argument" label="Passing the private key as argument (unsafe)">
+
+```bash
+ash avalanche subnet create -w -p PrivateKey-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+```
+
+  </TabItem>
+</Tabs>
+
+```bash
+Waiting for transaction to be accepted...
+Subnet created! (Tx ID: 'GQE4XUWgCR8ZryvhNdaCoyqVi25YNyh2nxyAjyCB3jXFL6gbk')
+Subnet 'GQE4XUWgCR8ZryvhNdaCoyqVi25YNyh2nxyAjyCB3jXFL6gbk':
+  Type: Permissioned
+  Control keys: ["P-local18jma8ppw3nhx5r4ap8clazz0dps7rv5u00z96u"]
+  Threshold:    1
+  Blockchains list (0): []
+  Validators list (0): []
+```
+
+:::note
+We used the `-w` flag to wait for the transaction to be accepted.
+:::
+
+## Create the Subnet EVM blockchain configuration
+
+Avalanche CLI provides an awesome wizard to create the Subnet EVM blockchain genesis configuration. Follow the official tutorial: [Create Your Subnet Configuration](https://docs.avax.network/subnets/build-first-subnet#create-your-subnet-configuration).
+
+This will create a `genesis.json` file at `~/.avalanche-cli/subnets/$YOUR_SUBNET` where `$YOUR_SUBNET` is the name you chose for your Subnet (e.g. `testSubnetEVM`):
+
+```bash
+cat ~/.avalanche-cli/subnets/testSubnetEVM/genesis.json
+```
+
+```json
+{
+  "config": {
+    "chainId": 11111,
+    "feeConfig": {
+      [...]
+    }
+  }
+}
+```
+
+## Create the Subnet EVM blockchain
+
+### Encode the genesis data
+
+At blockchain creation, we need to provide the byte-encoded genesis data. We use the `vm encode-genesis` command and store the output to a temporary JSON file:
+
+```bash
+# Encode the genesis data
+ash avalanche vm encode-genesis \
+  "~/.avalanche-cli/subnets/testSubnetEVM/genesis.json" --json > /tmp/encoded-genesis.json
+
+# Display the encoded genesis data
+cat /tmp/encoded-genesis.json
+```
+
+```json
+{ "genesisBytes": "0x7b22636f6e666967223a..." }
+```
+
+### Submit the blockchain creation transaction
+
+Use the `blockchain create` command to create the Subnet EVM blockchain in the Subnet we created earlier:
+
+<Tabs>
+  <TabItem value="env var" label="Using AVALANCHE_PRIVATE_KEY" default>
+
+```bash
+ash avalanche blockchain create testSubnetEVM \
+  --subnet-id GQE4XUWgCR8ZryvhNdaCoyqVi25YNyh2nxyAjyCB3jXFL6gbk \
+  --vm-type SubnetEVM \
+  --genesis-file /tmp/encoded-genesis.json -w
+```
+
+  </TabItem>
+  <TabItem value="argument" label="Passing the private key as argument (unsafe)">
+
+```bash
+ash avalanche blockchain create testSubnetEVM \
+  -p PrivateKey-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX \
+  --subnet-id GQE4XUWgCR8ZryvhNdaCoyqVi25YNyh2nxyAjyCB3jXFL6gbk \
+  --vm-type SubnetEVM \
+  --genesis-file /tmp/encoded-genesis.json -w
+```
+
+  </TabItem>
+</Tabs>
+
+```bash
+Waiting for transaction to be accepted...
+Blockchain created! (Tx ID: 'i2ZXw8NbVFxUK3kMjpwPtF2T3mtyoc6VWuH41rMfbc1NWE1Pb')
+Blockchain 'testSubnetEVM':
+  ID:      i2ZXw8NbVFxUK3kMjpwPtF2T3mtyoc6VWuH41rMfbc1NWE1Pb
+  VM type: SubnetEVM
+  RPC URL:
```

Comment:
Maybe explain why RPC URL is empty here?

Reply:
This is because the CLI is stateless. I guess I could just remove it from the output to avoid confusion.